src/server/app.js

```diff
@@ -57,6 +57,7 @@ if (log.level !== LogLevel.SILENT) {
 
 app.use(favicon(path.join(__dirname, '..', 'client', 'public', 'favicon.ico')));
 app.use(bodyParser.json({ limit: '50mb' }));
+app.use(bodyParser.raw({ limit: '50mb', type: ['application/xml'] }));
```

Comment:
This can also be removed, because we are not using xml.

Reply:
Done.